tests/terraform/graph/checks_infra/attribute_solvers/jsonpath_equals_solver/test_solver.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+from tests.terraform.graph.checks_infra.test_base import TestBaseSolver
```

Comment:
I'm not 100% sure, if the checks you add are also cleaned up, so they don't trigger extra results. in other tests, need to double check